src/main/java/org/folio/consortia/service/FolioTenantService.java

```diff
@@ -59,4 +80,14 @@ protected boolean tenantExists() {
       )
     );
   }
+
+  private void createOriginalTenantIdCustomField() {
+    try (var ignored = new FolioExecutionContextSetter(contextHelper.getSystemUserFolioExecutionContext(folioExecutionContext.getTenantId()))) {
+      if (ObjectUtils.isNotEmpty(customFieldService.getCustomFieldByName(ORIGINAL_TENANT_ID))) {
```

Comment:
Won't there NotFound error be thrown if field does not exist?

Reply:
Agree, @SinghAdes lets not throw exception , just return null

Reply:
fixed